pkg/manager-nnf/persistent.go

```diff
@@ -90,9 +90,12 @@ func (*DefaultPersistentController) DeletePersistentObject(obj PersistentObjectA
 	if err != nil {
 		return err
 	}
-	defer ledger.Close()
```

Comment:
The method above this, UpdatePersistentObject(), also has this `defer Close()` which can happen on error. Should that defer be removed as well?